src/app/common/stores/applications/Distribution/Wallet.store.ts

```diff
@@ -169,9 +249,25 @@ class WalletStore extends BaseStore implements Wallet {
    * @param networkId: any
    */
   async networkHandler(networkId: any) {
-    console.log('networkId: ', networkId);
-    this.networkId = networkId;
-    this.wrongNetwork = networkId !== 1 && networkId !== undefined;
+    const notIncludeMainnet = NETWORKS_DATA.every((network: NetworkDataType) => {
```

Comment:
Can you pls find all similar places like this and extract the logic to some helpers like `haveNoMainnet(networkId)`
Just find in both PRs and in regular webapp similar places in each wallet store and move to helpers.
Otherwise I see that it just using the same lines over and over again.

Also it relates to how wallet parses network ID from hex etc